src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -7,6 +7,7 @@ import DescriptionBox from './DescriptionBox';
 import ContributionBar from './ContributionBar';
 import styles from './styles.module.scss';
 import nycMapBackground from './images/nyc_3.png';
+import sclFlyerEnglish from './images/scl-flyer-english.png'
```

Comment:
@nanxiy should we also include the Chinese version here? Maybe an English speaker donor knows a Chinese merchant and might not know the click the Chinese toggle at the top